docs/README.md

```diff
@@ -2,17 +2,19 @@
 
 ## Overview
 
-The `docs` folder contains end-to-end documentation on Kyma and the following components:
+The `docs` folder contains end-to-end documentation on Kyma and its components:
 
+* [API Gateway](api-gateway/docs)
 * [Application Connector](application-connector/docs/)
+* [Authorization and Authentication](authorization-and-authentication/docs/)
 * [Event Bus](event-bus/docs/)
+* [Kyma](kyma/docs)
```

Comment:
I would prefer to emphasise this part a bit more as this is a starting point really for the whole documentation, and it should be clear this is a first folder to look into for overarching documentation